### DISCUSSION TITLE
How to deal with in-treatment keys?

### DISCUSSION THREAD

**Discussion by flsimoes** [plazi/community#84] [Q&A]
How should we deal with keys that are part of a treatment?

I've seen a trend in Insecta Mundi where authors tend to add snipets of keys (quite often extracted from previous publications) within a taxon's description:

![image](https://user-images.githubusercontent.com/3343086/74652861-bba63480-5165-11ea-822b-3fb2c39d8928.png)

**Comment by myrmoteras** [comment]
within treatment as subSubSection type=key

**Comment by flsimoes** [comment]
Thanks Donat!
